numpy/distutils/system_info.py

```diff
@@ -1939,6 +1878,58 @@ def calc_info(self):
                     include_dirs=incl_dirs)
         self.set_info(**info)
 
+class accelerate_info(system_info):
```

Comment:
Maybe `OSX_AccelerateInfo`?

Reply:
Assuming it isn't currently used publicly.

Reply:
Note that this would be a break from the naming convention in the file.
Should I follow the usual CamelCase regardless?

Reply:
Nah, let's not worry about it. Someday this will all need to be rewritten for clarity, but we can live with it a bit longer.